game_shared/ff/ff_gamerules.cpp

```diff
@@ -181,6 +181,11 @@ ConVar mp_prematch( "mp_prematch",
 					FCVAR_NOTIFY|FCVAR_REPLICATED,
 					"delay before game starts" );
 
+ConVar mp_armorstrip( "mp_armorstrip",

```

Comment:
I wonder if we should call this mp_friendlyfire_armorstrip? so when changing friendlyfire you get the intellisense and searching for friendlyfire brings this up.